sdk/python/tests/test_feature_store.py

```diff
@@ -63,7 +64,7 @@ def feature_store_with_gcs_registry():
     bucket.blob("registry.db")
 
     return FeatureStore(
-        config=RepoConfig(
+        config=create_repo_config(
```

Comment:
Is the idea that users would also be calling `create_repo_config`?

Reply:
Yes - RepoConfig can't know upfront what the online_store type will be so it's wrapped by `create_repo_config` to keep the layers separate.